readme.txt

```diff
@@ -0,0 +1,13 @@
+=== Give - Addon Boilerplate ===
```

Comment:
If this isn't being submitted to the plugin repository then this file is not necessary.